cmd/wallet-web/src/store/modules/user.js

```diff
@@ -280,11 +289,113 @@ export default {
             'gnap-user-subject': gnapOpts?.subjectId || '',
           });
 
+          let profileOpts = rootGetters.getProfileOpts;
+
+          Object.assign(profileOpts, { userConfig: { accessToken: gnapOpts?.accessToken || '' } });
+
+          await axios
+            .get(rootGetters.hubAuthURL + '/gnap/bootstrap', {
+              headers: { Authorization: `GNAP ${gnapOpts?.accessToken}` },
+            })
+            .then((resp) => {
+              let { data } = resp;
+
+              // TODO to be removed after universal wallet migration
+              if (opts.storageType === 'edv') {
+                const edvVaultURL = data.data.edvVaultURL;
+                const edvVaultID = data.data.userEDVVaultID;
+
+                opts.edvVaultID = edvVaultID;
+                // TODO this property is not returned from the bootstrap data - remove if not needed
+                opts.edvCapability = data.data.edvCapability;
+              }
+
+              // TODO to be removed after universal wallet migration
+              if (opts.kmsType === 'webkms') {
+                opts.opsKeyStoreURL = data.data.opsKeyStoreURL;
+                opts.edvOpsKIDURL = data.data.edvOpsKIDURL;
+                opts.edvHMACKIDURL = data.data.edvHMACKIDURL;
+              }
+
+              // TODO to be removed after universal wallet migration
+              // TODO this property is not returned from the bootstrap data - remove if not needed
+              opts.authzKeyStoreURL = data.data.authzKeyStoreURL;
+              // TODO this property is not returned from the bootstrap data - remove if not needed
+              // it is not even defined in the defaultAgentStartupOpts
+              // opts.userConfig = data.data.userConfig;
+              // TODO this property is not returned from the bootstrap data - remove if not needed
+              opts.opsKMSCapability = data.data.opsKMSCapability;
+
+              Object.assign(profileOpts, { bootstrap: data });
+
+              commit('updateAgentOpts', opts, { root: true });
+              commit('updateProfileOpts', profileOpts, { root: true });
+            })
+            .catch((err) => {
+              console.log('error fetching bootstrap data BEFORE Agent Init: errMsg=', err);
```

Comment:
suggestion: use console.error or console.warn - so that it gets highlighted in console logs